apps/lacework-accounts/awk_eks_audit_log/main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
```

Comment:
Quick note about location; I'm wondering if this is a better fit under the util folder structure? https://github.com/lacework-dev/detc-resources/tree/main/util